### ISSUE TITLE
Packages not recognized

### ISSUE THREAD

**Issue by oaustegard** [federico-terzi/espanso#1000]
**Describe the bug**
While Packages seem to install correctly and are shown after installation using `espanso packages list` the matches in the package do not work, nor show in the search bar

**To Reproduce**
May be hard to reproduce - I previously had 0.7.3 installed and had a few issues getting it cleanly uninstalled.  2.0 works with regular matches (both in base.yaml and any custom named ones) in the \match folder.

I believe the folder setup is as expected:? 
``` powershell
PS C:\Users\austegard.oskar> espanso path
Config: C:\Users\austegard.oskar\AppData\Roaming\espanso  #(matches are in \match here as normal)
Packages: C:\Users\austegard.oskar\AppData\Local\espanso\packages  #(none of the matches here work)
Runtime: C:\Users\austegard.oskar\AppData\Local\espanso
```

**Expected behavior**
Normal Package behavior - matches match and expansions happen...

**Logs**
``` powershell
PS C:\Users\austegard.oskar\AppData\Local\espanso> more espanso.log
16:53:18 [daemon(26712)] [INFO] reading configs from: "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso"
16:53:18 [daemon(26712)] [INFO] reading packages from: "C:\\Users\\austegard.oskar\\AppData\\Local\\espanso\\packages"
16:53:18 [daemon(26712)] [INFO] using runtime dir: "C:\\Users\\austegard.oskar\\AppData\\Local\\espanso"
16:53:18 [daemon(26712)] [INFO] watching for changes in path: "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso"
16:53:18 [daemon(26712)] [INFO] espanso version: 2.1.3-alpha
16:53:18 [daemon(26712)] [INFO] spawning the worker process...
16:53:18 [daemon(26712)] [INFO] binded to named pipe: \\.\pipe\espansodaemonv2
16:53:18 [worker(7560)] [INFO] reading configs from: "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso"
16:53:18 [worker(7560)] [INFO] reading packages from: "C:\\Users\\austegard.oskar\\AppData\\Local\\espanso\\packages"
16:53:18 [worker(7560)] [INFO] using runtime dir: "C:\\Users\\austegard.oskar\\AppData\\Local\\espanso"
16:53:18 [worker(7560)] [INFO] binded to named pipe: \\.\pipe\espansoworkerv2
16:53:18 [worker(7560)] [INFO] monitoring the status of the daemon process
16:53:18 [worker(7560)] [INFO] using Win32AppInfoProvider
16:53:18 [worker(7560)] [INFO] using Win32Source
16:53:18 [worker(7560)] [INFO] using Win32Injector
16:53:18 [worker(7560)] [INFO] using Win32Clipboard
PS C:\Users\austegard.oskar\AppData\Local\espanso>
```

**Setup information**
- OS: Windows 10 Enterprise
- Version: 2.1.3-alpha

**Comment by oaustegard** [comment]
Could this signify that the old version was not fully removed somehow? 
![image](https://user-images.githubusercontent.com/3268739/155416203-651626d0-f514-4748-8b50-ab4a0c6009ec.png)

**Comment by federico-terzi** [comment]
Hey @oaustegard,

Thanks for reporting this issue! When you have the chance, could you please try typing `#acfg#` and post the output here? That will give us some useful information to debug this further :)

> Could this signify that the old version was not fully removed somehow?

That's unlikely, but you can check the folder at `C:\Users\User\AppData\Local\espanso` and see if there's an `espanso.exe` file there, and remove it if so

Cheers :)

**Comment by oaustegard** [comment]
Here you go:
```
[espanso config: "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso\\config\\default.yml"]

backend: Auto
enable: true
paste_shortcut: None
inject_delay: None
key_delay: None
apply_patch: true
word_separators: [" ", ",", ".", "?", "!", "\r", "\n", "\u{16}"]

preserve_clipboard: true
clipboard_threshold: 100
disable_x11_fast_inject: false
pre_paste_delay: 100
paste_shortcut_event_delay: 10
toggle_key: Some(RightCtrl)
auto_restart: true
restore_clipboard_delay: 300 
backspace_limit: 5
search_trigger: Some(":search")
search_shortcut: Some("CTRL+SHIFT+E")
keyboard_layout: None

show_icon: true
show_notifications: true
secure_input_notification: true

x11_use_xclip_backend: false
win32_exclude_orphan_events: true
win32_keyboard_layout_cache_interval: 2000

match_paths: [
    "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso\\match\\greek.yml",
    "C:\\Users\\austegard.oskar\\AppData\\Roaming\\espanso\\match\\base.yml",
]
```

**Comment by federico-terzi** [comment]
@oaustegard Thanks! There seems to be a problem with the migration of packages, I'll investigate :)

In the meantime, try to move the `C:\Users\austegard.oskar\AppData\Local\espanso\packages` directory inside `C:\Users\austegard.oskar\AppData\Roaming\espanso\match`, ending up with: `C:\Users\austegard.oskar\AppData\Roaming\espanso\match\packages`. That should make them work until we figure out the cause :)

**Comment by oaustegard** [comment]
Makes sense, good to know (already moved the match file for the package I use into match, so same effect). Begs the question - why have Packages in a separate folder path at all?

**Comment by federico-terzi** [comment]
@oaustegard Nice, thanks!

> Begs the question - why have Packages in a separate folder path at all?

That's a good question! The reason is that I wanted to keep Espanso backward compatible with the "legacy" locations (pre-v2) to avoid breaking existing installations. Some of the early design choices weren't optimal for sure, in retrospect. Hopefully, most of them will gradually go away, as if you install the v2 from scratch, packages are already inserted inside the `match` directory :)

**Comment by oaustegard** [comment]
Thanks! Does that mean there’s a setting I can change so as to have future packages install to the match directory?

**Comment by federico-terzi** [comment]
> Thanks! Does that mean there’s a setting I can change so as to have future packages install to the match directory?

Sure! Just move the `packages` folder as described here

> In the meantime, try to move the C:\Users\austegard.oskar\AppData\Local\espanso\packages directory inside C:\Users\austegard.oskar\AppData\Roaming\espanso\match, ending up with: C:\Users\austegard.oskar\AppData\Roaming\espanso\match\packages. That should make them work until we figure out the cause :)

And delete the previous at ` C:\Users\austegard.oskar\AppData\Local\espanso\packages`

Espanso should automatically pick up the right one after a restart :) (you can check the logs to make sure)

**Comment by smeech** [comment]
I'm going to close this as I don't think we'll be attending to migration issues now that almost everyone is using v2.
